From 33482f02a766adccce6fd7574073b913351e86aa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Mon, 19 Jul 2010 18:47:22 +0000
Subject: [PATCH] gitignore: Ignore the new /git-remote-{ext,fd} helpers
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the top-level .gitignore to include the new
/git-remote-{ext,fd} helpers. They were added as part of the
il/rfc-remote-fd-ext, but .gitignore wasn't proprely updated in that
series..

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 .gitignore |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/.gitignore b/.gitignore
index 2cb4e8e..eda6fbc 100644
--- a/.gitignore
+++ b/.gitignore
@@ -108,6 +108,8 @@
 /git-relink
 /git-remote
 /git-remote-curl
+/git-remote-ext
+/git-remote-fd
 /git-remote-http
 /git-remote-https
 /git-remote-ftp
-- 
1.7.2.rc3.125.g94e09.dirty

